src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -354,9 +347,6 @@ void AutomatedRssDownloader::clearRuleDefinitionBox()
     ui->comboAddPaused->clearEditText();
     ui->comboAddPaused->setCurrentIndex(-1);
     updateFieldsToolTips(ui->checkRegex->isChecked());
-    updateMustLineValidity();
-    updateMustNotLineValidity();
-    updateEpisodeFilterValidity();
```

Comment:
These lines are necessary for ensuring the text fields are re-checked for validity when all the fields are set to empty/default values e.g. if zero or multiple rules are selected. Otherwise if the old selection had an invalid entry these won't be updated correctly.